cmd/tellor/main.go

```diff
@@ -106,20 +116,22 @@ func AddDBToCtx(remote bool) error {
 	os.RemoveAll(cfg.DBFile)
 	DB, err := db.Open(cfg.DBFile)
 	if err != nil {
-		return err
+		return fmt.Errorf("Error opening DB: %v\n", err)
```

Comment:
using errors.Wrap is better here:

```suggestion
		return errors.Wrap(err,"opening the DB")
```

Reply:
I'll leave this as is right now because this same thing is used all over the code. But if it's worth the time, I can do another PR to adjust those all  in one batch. What do you think?

Reply:
I think it is fine to update gradually, since you are changing the code here anyway why not just use the proper form and the rest can do in follow up PRs.

Reply:
I think it'll be easier to do it in a new one This way we won't add a lot of new changes to this already big Pr!